### DISCUSSION TITLE
Webdriver: request failed due to response error: unknown error when running WebdriverIO tests in Safari 16

### DISCUSSION THREAD

**Discussion by anilpkumar18** [webdriverio/webdriverio#10840] [Q&A]
The Tests are running fine in other browsers. When running the test in Safari browser version 16 (Mac OS - Ventura 13.3) I am getting below error

webdriver: request failed due to response error: unknown error Request failed with status 500 due to unknown error unknown error: unknown error at processTicksAndRejections (node:internal/process/task_queues:96:5)

Browser is initiated and Login works fine. Post login, getting this error when validating the tester or clicking on element etc.

Capabilities:

platformName:'Mac', browserName: 'safari', browserVersion: '16.4',

export const config: Options.Testrunner = { protocol: 'http', hostname: 'localhost', path: '/', port: 4444, services: [ ['safaridriver', { outputDir: './logs', logFileName: 'wdio-safaridriver.log' }] ], autoCompileOpts: { autoCompile: true, tsNodeOpts: { transpileOnly: true, project: './tsconfig.json' } }

**Versions: **"wdio-safaridriver-service": "^2.1.1", "webdriverio": "^7.32.3" "selenium-webdriver": "^4.10.0", "@wdio/cli": "^7.32.3", "@wdio/cucumber-framework": "^7.32.3", "@wdio/local-runner": "^7.32.3", "npm": "^8.7.0",

I have the option enabled "Allow Remote Automation" in Safari browser.

Additionally it seems the issue when page is reloaded. Here post login , Home page is loaded and when finding elements in the Home page, it throws error.

Please note the tests are working fine in Chrome, Firefox and Edge browser.

**Comment by christian-bromann** [comment]
@anilpkumar18 thanks for raising the issue.

Can you provide a minimal reproducible example?

**Reply by anilpkumar18** [reply]
@christian-bromann 

Apologies. Unfortunately I cannot provide access to application to reproduce. Just to give front end technologies user are Angular 5 with Canvas layer and Microfrontend. 

The tests were running till Mac OS Mojave and Safari 13. Is there any open issue on the above technologies with webdriver running in Safari 16?

**Reply by christian-bromann** [reply]
Providing a reproducible example doesn't mean giving access to your application it means providing a little project that allows me to reproduce this.
